tst/schedulers/test_multi_fidelity_kde.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+
+from syne_tune.optimizer.schedulers.searchers.multi_fidelity_kde_searcher import MultiFidelityKernelDensityEstimator
```

Comment:
Can you add a test in the general api tests? https://github.com/awslabs/syne-tune/blob/main/tst/schedulers/test_schedulers_api.py#L44